ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/OpenSSL/tasks/main.yml

```diff
@@ -4,57 +4,57 @@
   tags: openssl
 
-- name: Unpack OpenSSL-1.1.1e for installation
+- name: Unpack OpenSSL-1.1.1f for installation
   win_shell: |
     cd C:\temp
-    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1e.tar.gz
-    C:\7-Zip\7z.exe x C:\temp\openssl-1.1.1e.tar
+    C:\7-Zip\7z.exe x C:\temp\OpenSSL-1.1.1f.tar.gz
+    C:\7-Zip\7z.exe x C:\temp\OpenSSL-1.1.1f.tar
   when: (not openssl32_installed.stat.exists) or (not openssl64_vs2013_installed.stat.exists) or (not openssl64_vs2017_installed.stat.exists)
   tags: openssl
 
-- name: Install OpenSSL-1.1.1e 32-bit (VS2013)
-  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vsvars32.bat && cd C:\temp\openssl-1.1.1e && perl C:\temp\openssl-1.1.1e\Configure VC-WIN32 --prefix=C:\openjdk\OpenSSL-1.1.1e-x86_32-VS2013 && nmake install > C:\temp\openssl32.log && nmake -f makefile clean
+- name: Install OpenSSL-1.1.1f 32-bit (VS2013)
+  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vsvars32.bat && cd C:\temp\OpenSSL-1.1.1f && perl C:\temp\OpenSSL-1.1.1f\Configure VC-WIN32 --prefix=C:\openjdk\OpenSSL-1.1.1f-x86_32-VS2013 && nmake install > C:\temp\openssl32.log && nmake -f makefile clean
   args:
     chdir: 'C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\Tools'
     executable: cmd
   when: (not openssl32_installed.stat.exists)
   tags: openssl
 
-- name: Install OpenSSL-1.1.1e 64-bit (VS2013)
-  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vcvarsall.bat AMD64 && cd C:\temp\openssl-1.1.1e && perl C:\temp\openssl-1.1.1e\Configure VC-WIN64A --prefix=C:\openjdk\OpenSSL-1.1.1e-x86_64-VS2013 && nmake install > C:\temp\openssl64.log && nmake -f makefile clean
+- name: Install OpenSSL-1.1.1f 64-bit (VS2013)
+  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vcvarsall.bat AMD64 && cd C:\temp\OpenSSL-1.1.1f && perl C:\temp\OpenSSL-1.1.1f\Configure VC-WIN64A --prefix=C:\openjdk\OpenSSL-1.1.1f-x86_64-VS2013 && nmake install > C:\temp\openssl64.log && nmake -f makefile clean
   args:
     chdir: 'C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC'
     executable: cmd
   when: (not openssl64_vs2013_installed.stat.exists)
   tags: openssl
 
-- name: Install OpenSSL-1.1.1e 64-bit (VS2017)
-  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vcvarsall.bat AMD64 && cd C:\temp\openssl-1.1.1e && perl C:\temp\openssl-1.1.1e\Configure VC-WIN64A --prefix=C:\openjdk\OpenSSL-1.1.1e-x86_64-VS2017 && nmake install > C:\temp\openssl64.log && nmake -f makefile clean
+- name: Install OpenSSL-1.1.1f 64-bit (VS2017)
+  win_shell: set PATH=C:\Strawberry\perl\bin;C:\openjdk\nasm-2.13.03;%PATH% && .\vcvarsall.bat AMD64 && cd C:\temp\OpenSSL-1.1.1f && perl C:\temp\OpenSSL-1.1.1f\Configure VC-WIN64A --prefix=C:\openjdk\OpenSSL-1.1.1f-x86_64-VS2017 && nmake install > C:\temp\openssl64.log && nmake -f makefile clean
```

Comment:
Needs to create a new log for OpenSSL-64 bit (VS2017), as it will current just add onto the OpenSSL-64bit (VS2013) log.

Reply:
does the log not get cleaned up each time?

Reply:
Nope, stays in `C:\temp\`

Reply:
@Willsparker Does it add onto or overwrite? We can have different logs for the different VS (2013 & 2017) but if we will be keeping the logs at the end in the event they need to be looked over, then I guess we don't need a cleanup job for them at the end.

Reply:
Looking at some machines that have recently been setup, it looks like it just overwrites the file, so we've lost the log for 64-bit VS2013. I think it's worth keeping the logs around, if not for anything else, then because we can compare logs if it succeeds building on one system, but fails on another :-)